helm-chart/templates/cronjob.yaml

```diff
@@ -0,0 +1,111 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  labels:
+    app: popeye-reports
+  name: popeye-reports
+  namespace: {{ .Values.namespace }}
+spec:
+  schedule: {{ .Values.schedule }}
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: {{ .Values.failedJobsHistoryLimit | default "20" }}
+  successfulJobsHistoryLimit: {{ .Values.successfulJobsHistoryLimit | default "20" }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: popeye-reports
+        spec:
+          containers:
+          - image: "{{ .Values.image.web.name }}:{{ .Values.image.web.tag }}"
```

Comment:
I believe you would like to serve html reports via this nginx:

This is a CronJob which executes a Job periodically, the Job schedules a Pod with this container, you basically create a Job with an endless runtime, since nginx serves its content and wont stop until aborted.
Even if this was the correct approach, Popeye (and therefore itss Helm Chart) is not responnsible for serving HTML content.
Also in k8s/popeye/ is nothing related to nginx.

Remove the whole nginx-stuff, since again it has nothing to do with Popeye's actual purpose - reporting.